spec/unit/shoulda/matchers/action_controller/callback_matcher_spec.rb

```diff
@@ -47,10 +47,90 @@ def match
       end
     end
 
+    context "qualified with only" do
+      let(:action) { :index }
+      let(:multiple_actions) { [:index, :show] }
+      let(:other_action) { :other_action }
+
+      context "and when a #{kind} hook is in place with the qualifier" do
+        it "accepts" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.